### PR TITLE
Fix capitalization of page titles

### LINKED ISSUE #82: Fix capitalization of page titles
Especially the VILLASnode hook pages are all different.

### PR DESCRIPTION
**Pull request by al3xa23**
Addressed https://github.com/VILLASframework/documentation/issues/82#issue-2144408531

### CODE REVIEW

File: docs/node/nodes/exec.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
+---
+sidebar_label: Execution
+---
+
 ---
 hide_table_of_contents: true
 ---
 
-# Sub-process Execution
+# Execution of Sub-process
```
Comment (stv0g) [inline docs/node/nodes/exec.md]:
```suggestion
# Execution of Sub-processes
```

File: docs/node/hooks/shift_ts.md [hunk 2]
```diff
@@ -2,7 +2,10 @@
 hide_table_of_contents: true
 ---
 
-# Shift timestamps
+# Shift Timestamps
+
+The hook `shift_ts` is designed to shift timestamps by applying a specified time offset.
```
Comment (stv0g) [inline docs/node/hooks/shift_ts.md]:
```suggestion
The hook `shift_ts` is designed to shift timestamps by applying a specified offset in seconds. 
```

File: docs/node/hooks/shift_seq.md [hunk 3]
```diff
@@ -2,7 +2,9 @@
 hide_table_of_contents: true
 ---
 
-# Shift sequence numbers
+# Shift Sequence Numbers
+
+The `shift_seq` hook is designed to modify the sequence number of samples.
```
Comment (stv0g) [inline docs/node/hooks/shift_seq.md]:
```suggestion
The `shift_seq` hook is designed to modify the sequence number of samples with an offset.
```

File: docs/node/hooks/rms.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 hide_table_of_contents: true
 ---
 
-# RMS signals
+# RMS Signals
```
Comment (stv0g) [inline docs/node/hooks/rms.md]:
```suggestion
# Root-Mean-Square Calculation
```

File: docs/node/hooks/jitter_calc.md [hunk 5]
```diff
@@ -4,6 +4,8 @@ hide_table_of_contents: true
 
 # Calculate Jitter
 
+The `jitter_calc` hook is designed to calculate the jitter, mean, and variance between GPS and NTP timestamps.
```
Comment (stv0g) [inline docs/node/hooks/jitter_calc.md]: I dont think it has any notion of GPS and NTP timestamps..
Can you check the code? I believe its calculating jitter based on inter-sample timing?

File: docs/node/hooks/index.md [hunk 6]
```diff
@@ -53,16 +53,17 @@ Currently, the following _node-hook-types_ are supported:
...
+| [`round`](round.md)             |           | yes       | yes        | yes  | stable     | Round floating point and complex signals to decimal digits |
 | [`scale`](scale.md)             |           | yes       | yes        | yes  | stable     | Scale signals by a factor and add offset |
 | [`shift_seq`](shift_seq.md)     |           | yes       | yes        |      | stable     | Shift sequence number of samples |
 | [`shift_ts`](shift_ts.md)       |           | yes       | yes        |      | stable     | Shift timestamps of samples |
 | [`skip_first`](skip_first.md)   |           | yes       | yes        | yes  | stable     | Skip the first samples |
 | [`stats`](stats.md)             |           | yes       |            |      | stable     | Collect statistics for the current path |
 | [`ts`](ts.md)                   |           | yes       | yes        | yes  | stable     | Overwrite origin timestamp of samples with receive timestamp |
-| [`print`](print.md)             |           | yes       | yes        | yes  | stable     | Print the message to stdout |
+
```
Comment (stv0g) [inline docs/node/hooks/index.md:66]: Are there now two empty lines? Please remove one

File: docs/node/hooks/ebm.md [hunk 7]
```diff
@@ -2,7 +2,9 @@
 hide_table_of_contents: true
 ---
 
-# Energy Based Metric calculation
+# Energy Based Metric Calculation
+
+The `ebm` hook calculates the energy consumption based on voltage and current samples over time.
```
Comment (stv0g) [inline docs/node/hooks/ebm.md]:
```suggestion
The `ebm` hook calculates the energy of a signal based on voltage and current product.
```

File: docs/node/hooks/dp.md [hunk 8]
```diff
@@ -2,7 +2,9 @@
 hide_table_of_contents: true
 ---
 
-# Dynamic Phasor calculation / DFT
+# Dynamic Phasor (DP) Calculation / DFT
```
Comment (stv0g) [inline docs/node/hooks/dp.md]:
```suggestion
# Dynamic Phasor (DP) Calculation / Discrete Fourier Transform (DFT)
```

**Review by stv0g — CHANGES_REQUESTED** [review]
Thanks :) Nice work. I only have some minor changes requested.

File: docs/node/hooks/jitter_calc.md (hunk 5 above)
Comment (al3xa23) [reply docs/node/hooks/jitter_calc.md]: Yes, you are right. I changed the docu accordingly.

**Review by stv0g — APPROVED** [review]

**Pull request merged**